guides/source/active_support_core_extensions.md

```diff
@@ -837,6 +839,25 @@ In the previous example the macro generates `avatar_size` rather than `size`.
 
 NOTE: Defined in `active_support/core_ext/module/delegation.rb`
 
+#### `delegate_missing_to`
+
+Imagine you would like to delegate everything missing from the `User` object,
+to the `Profile` one. The `delegate_missing_to` macro lets you implement this
+in a breeze:
+
+```ruby
+class User < ApplicationRecord
+  has_one :profile
+
+  delegate_missing_to :profile
+end
+```
+
+The target can be anything callable within the object, e.g. instance variables,
+methods, constants, etc.
+
+NOTE: Defined in `active_support/core_ext/module/delegation.rb`
```

Comment:
Just dropped it in a sentence. If you think a warning, like the one in https://github.com/rails/rails/pull/29268#discussion_r118934489 works better, ping me and I'll change it.